### ISSUE TITLE
barcode scanner  issue

### ISSUE THREAD

**Issue by FizaAnsar** [capacitor-community/barcode-scanner#285]
**Describe the bug**
After scanning it does not navigate to another URL or another component. just like if we want login functionality using qr code after successfully scanning it must be go to the home page when Login Successful but it does not work

**To Reproduce**

4. See the error

**Expected behavior**

**Screenshots**

**Version**
[e.g. v1.0.0]

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by Abk8406** [comment]
@FizaAnsar  Yeah I'll give you a solution, I also faced that issue last time the same issue came.
